packages/gatsby-dev-cli/src/watch.js

```diff
@@ -17,7 +17,7 @@ function watch(root, packages, { scanOnce, quiet }) {
     const ignoreRegs = [
       /[\/\\]node_modules[\/\\]/i,
       /\.git/i,
-      new RegExp(`${p}[\\/]src[\\/]`, `i`),
+      new RegExp(`${p}[\\/\\\\]src[\\/\\\\]`, `i`),
```

Comment:
Isn't \\ enough already? I understood that / doesn't have to be escaped in a string passed to RegExp so [\\/] means \ or /.